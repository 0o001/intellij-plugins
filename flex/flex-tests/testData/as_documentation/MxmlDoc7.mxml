<?xml version="1.0"?>
<mx:VBox xmlns="http://ns.adobe.com/mxml/2009" xmlns:mx="library:adobe/flex/halo" >
        <!--- 
         Comment for button
        --> 
	<mx:Button id="myButton" label="This button has comment"/>
  <mx:Script>
    function foo() {
      myBut<caret>ton
    }
  </mx:Script>
</mx:VBox>

