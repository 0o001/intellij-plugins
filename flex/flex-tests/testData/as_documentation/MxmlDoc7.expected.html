<div class='definition'><pre>myButton</pre></div><div class='content'><p>         Comment for button</div><table class='sections'></table>